frontend/packages/dev-console/src/components/topology/TopologyPage.tsx

```diff
@@ -56,6 +65,39 @@ export function renderTopology({ loaded, loadError, data }: RenderProps) {
   );
 }
 
+const LoadPage: React.FC<LoadPageProps> = ({
+  application,
+  knative,
+  namespace,
+  projects,
+  cheURL,
+}) => {
+  if (projects.loaded) {
+    if (_.isEmpty(projects.data)) {
+      return (
+        <EmptyState
+          title="Add"
```

Comment:
I believe the title of this page is supposed to be `Topology`.

Reply:
Fixed